extension/js/popup.js

```diff
@@ -1,14 +1,84 @@
 /* global browser */
 
+function getOnboardingPanels() {
+  return {
+    "panel1": {
+      "imgSrc": "tip1-icon.svg",
+      "tipHeadline": "Welcome!",
+      "tipBody": "When the Firefox Relay icon appears on a website, select it to generate a new relay address."
+
+    },
+    "panel2": {
+      "imgSrc": "tip2-icon.svg",
+      "tipHeadline": "Don't see the icon?",
```

Comment:
nit: `Don't` has a straight quote, whereas the `You’re` below on L24 is a smart quote.